gateway-service-impl/build.gradle.kts

```diff
@@ -11,28 +11,27 @@ tasks.test {
 dependencies {
   api(project(":gateway-service-api"))
 
-  implementation("org.hypertrace.core.query.service:query-service-client:0.5.2")
-  implementation("org.hypertrace.core.attribute.service:attribute-service-client:0.9.3")
-  implementation("org.hypertrace.entity.service:entity-service-client:0.5.6")
-  implementation("org.hypertrace.entity.service:entity-service-api:0.5.6")
-  implementation("org.hypertrace.core.grpcutils:grpc-context-utils:0.3.4")
-  implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.28")
+  implementation("org.hypertrace.core.query.service:query-service-client:0.6.2")
+  implementation("org.hypertrace.core.attribute.service:attribute-service-client:0.12.5")
+  implementation("org.hypertrace.entity.service:entity-service-client:0.8.1")
+  implementation("org.hypertrace.entity.service:entity-service-api:0.8.1")
+  implementation("org.hypertrace.core.grpcutils:grpc-context-utils:0.5.2")
+  implementation("org.hypertrace.core.serviceframework:platform-metrics:0.1.29")
 
   // Config
   implementation("com.typesafe:config:1.4.1")
 
   // Common utilities
   implementation("org.apache.commons:commons-lang3:3.10")
   implementation("org.apache.commons:commons-math:2.2")
-  implementation("com.google.protobuf:protobuf-java-util:3.13.0")
+  implementation("com.google.protobuf:protobuf-java-util:3.17.3")
 
-  // Needed by clusters snapshots
-  implementation("com.fasterxml.jackson.core:jackson-annotations:2.11.1")
-  implementation("com.fasterxml.jackson.core:jackson-databind:2.11.1")
+  implementation("com.fasterxml.jackson.core:jackson-annotations:2.12.4")
+  implementation("com.fasterxml.jackson.core:jackson-databind:2.12.4")
 
   testImplementation("org.junit.jupiter:junit-jupiter:5.7.0")
   testImplementation("org.mockito:mockito-core:3.9.0")
   testImplementation("org.mockito:mockito-inline:3.9.0")
   testImplementation("org.apache.logging.log4j:log4j-slf4j-impl:2.14.0")
-  testImplementation("io.grpc:grpc-netty:1.37.0")
+  testImplementation("io.grpc:grpc-netty:1.39.0")
```

Comment:
tests should not be using netty 🙄  but probably out of scope

Reply:
I actually tried removing it, but there was a unit test which failed, so I put it back

Reply:
Again, indicates issues - that means we're actually spinning up transport in a unit test, which we shouldn't be doing.